### PR TITLE
allow multiple config files/directories as command line arg

### PR DESCRIPTION
**Pull request by zzeekk**
small non-breaking changes to command line:
- allow multiple config files/directories, separated by comma
- make name optional and use feed-sel as name if not given

### CODE REVIEW

File: src/main/scala/io/smartdatalake/config/ConfigLoader.scala [hunk 1]
```diff
@@ -148,12 +149,12 @@ object ConfigLoader extends SmartDataLakeLogger {
    *
    * This is an internal method to create a utility data structure.
    *
-   * @param rootPath      root [[Path]] pointing to a configuration file or a directory from where the traversal starts.
+   * @param rootPaths     root [[Path]]'s pointing to a configuration file or a directory from where the traversal starts.
```
Comment (pgruetter) [inline src/main/scala/io/smartdatalake/config/ConfigLoader.scala]:
```suggestion
   * @param rootPaths     root [[Path]]s pointing to a configuration file or a directory from where the traversal starts.
```

File: src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala [hunk 2]
```diff
@@ -75,7 +75,7 @@ abstract class SmartDataLakeBuilder extends SmartDataLakeLogger {
 
   // read version from package manifest (not defined if project is executed in IntellJ)
   val appVersion: String = Option(getClass.getPackage.getImplementationVersion).getOrElse("develop")
-  val appName: String = getClass.getSimpleName.replaceAll("\\$$","") // remove $ from object name and use it as appName
+  val appType: String = getClass.getSimpleName.replaceAll("\\$$","") // remove $ from object name and use it as appName
```
Comment (pgruetter) [inline src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala]: Can you give the reasoning behind the variable name change appName to appType ? I see sometimes it's still appName.
Didn't check everything, but I guess a few comments / references like here would also need to be changed.

File: src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala [hunk 3]
```diff
@@ -167,10 +166,11 @@ abstract class SmartDataLakeBuilder extends SmartDataLakeLogger {
     logger.info(s"Application: ${appConfig.applicationName}")
     logger.info(s"Master: ${appConfig.master}")
     logger.info(s"Deploy-Mode: ${appConfig.deployMode}")
+    val appName = appConfig.applicationName.getOrElse(appConfig.feedSel)
```
Comment (pgruetter) [inline src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala]: Did you try with different regular expressions if everything still works? i.e when using ".*" as feed selector

File: src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala (hunk 2 above)
Comment (zzeekk) [reply src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala]: The applicationName is actionally given as parameter on the command line. It's confusing if we use it also here, so i was looking for a different name.
I'll check the other references.

Comment (zzeekk) [reply src/main/scala/io/smartdatalake/app/SmartDataLakeBuilder.scala]: No additional wrong references found for appName with text search.

**Pull request merged**